### DISCUSSION TITLE
Weird "Failed to mark outputs as cached" messages

### DISCUSSION THREAD

**Discussion by jakubmazanec** [vercel/turborepo#2716] [Help] [answered]
Lately, when using Turborepo, I've been seeing a lot of messages like `Failed to mark outputs as cached for @somepackage#sometask: rpc error: code = Unknown desc = timed out waiting for cookie` or `Failed to check if we can skip restoring outputs for @anotherpackage#anothertask: rpc error: code = Unknown desc = timed out waiting for cookie. Proceeding to check cache`, but otherwise Turborepo seems working fine. I can't google any info. Does anyone have an idea what causes this? Thanks.

**Comment by novrain** [comment, marked as answer]
I got this output messgae too.

add --no-daemon to turbo build

[https://codesti.com/issue/vercel/turbo/2790](https://codesti.com/issue/vercel/turbo/2790)

**Reply by jakubmazanec** [reply]
Thanks.

**Comment by anthonyshew** [comment]
For anyone landing here from search, this was a bug that we have since mitigated as far as we know! If you continue seeing this problem please open an issue!
